dotcom-rendering/src/web/experiments/tests/sign-in-gate-copy-test-variants.ts

```diff
@@ -0,0 +1,29 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateCopyTest: ABTest = {
+	id: 'SignInGateCopyTest',
```

Comment:
I think this should be kept the same as the ComponentID going to ophan which has been the convention to date or people might think they are different tests. I would suggest keeping the const name consistent too.

Reply:
Good idea, done